README

```diff
@@ -14,10 +14,16 @@ You have presumably obtained the metaconfig from the repository e.g.
 
   $ git clone github.com:perl5-metaconfig/metaconfig metaconfig
 
-Normally this directory and perl directory are next to each other
-so ../perl will get you to perl and ../perl/../metaconfig will get you
-back here. The current setup will require a symbolic link to the folder
-where Configure is generated in:
+When working with metaconfig you will generally have two git checkouts
+next to each other:  (1) this metaconfig checkout; and (2) a checkout of
+the Perl 5 source code in which you will generate a new Configure
+script. In this README, we will refer to these directories as the
+'metaconfig' directory and the 'perl' directory.
+
+Since these two directories are normally next to each other, so ../perl
+will get you to perl and ../perl/../metaconfig will get you back here.
+You should establish a symbolic link to the checkout in which Configure
+is generated such as this:
 
```

Comment:
I don't think the metaconfig symlink is actually needed; at least I didn't set it up in my environment and mconfig and mlint appear to work.